### ISSUE TITLE
When the PM feature is enabled, can PMLEN be set to 7 in sv48 mode?

### ISSUE THREAD

**Issue by chara811** [riscv/riscv-isa-manual#1702]
Regarding sv48 mode, there is no clear restriction on setting PMLEN to 7 in the PM specification

**Comment by gfavor** [comment]
Yes.  The PMM fields are WARL and nothing in the spec restricts which PMLEN values can be used with which translation modes.

**Comment by chara811** [comment]
Does PM function support when *atp=sv39?

**Comment by chara811** [comment]
The PM spec Section 3.5 describes configurations such as "PMLEN=7 and hgatp.MODE=sv57x4,PMLEN=16 and hgatp.MODE=sv57x4,"PMLEN=16 and hgatp.MODE=sv48x4." However, it does not list whether sv39x4 can be configured with PMLEN=16 or 7, and sv48x4 can be configured with PMLEN=7.Are these configurations that are not listed supported?

**Comment by gfavor** [comment]
That spec text does not limit support to the listed configs.  That text is specifying when one would need to execute an HFENCE.GVMA when changing henvcfg.PMM (and implicitly is specifying when an HFENCE.GVMA is not necessary).

**Comment by aswaterman** [comment]
Closed as answered.
